docs/reference/configuration/observations.rst

```diff
@@ -72,29 +72,29 @@ created as follows:
 This will create an observation of group oil production for the Brent
 group on 21th of august 2005. The observed value was 100 with a
 standard deviation of 5. The name SEP_TEST_2005 will be used as a
-label for the observation within the ERT and must be unique.
+label for the observation within ERT and must be unique.
 
 Date format YYYY-MM-DD (ISO 8601) is required.
 Other time formats, like DD/MM/YYYY or DD.MM.YYYY, are deprecated
-and its support will be removed in a future release. The date format
+and their support will be removed in a future release. The date format
 can also include hours and seconds: "YYYY-MM-DDTHH:mm:ss". When the
```

Comment:
The date format string seems to have some sort of special meaning in rst. How does this look rendered?

Reply:
![image](https://github.com/equinor/ert/assets/45088507/ed0ffef6-bcb9-46c8-9390-311149c2341c)

Reply:
:+1: